module/app/upgrades/v2/README.md

```diff
@@ -0,0 +1,9 @@
+# MERCURY UPGRADE
```

Comment:
💯  ok we can leave everything named v2 so long as we have some good docs explaining it.